src/main/java/domain/Ladder.java

```diff
@@ -0,0 +1,39 @@
+package domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import exception.Error;
+
+public class Ladder {
+	private final List<Level> levels;
+
+	private Ladder(List<Level> level) {
+		this.levels = level;
+	}
+
+	public static Ladder from(int height, int participantSize) {
+		validate(height);
+		return new Ladder(IntStream.range(0, height)
+			.mapToObj(o -> new Level(participantSize))
+			.collect(Collectors.toList()));
+	}
+
+	private static void validate(int height) {
+		if (height < 1 || height > 100)
+			throw new IllegalArgumentException(Error.HEIGHT_RANGE_FROM_1_TO_100.getMessage());
```

Comment:
```suggestion
		if (height < 1 || height > 100) {
			throw new IllegalArgumentException(Error.HEIGHT_RANGE_FROM_1_TO_100.getMessage());
		}
```

전체적인 컨벤션은 PR 올리기 전 확인해주세요!

Reply:
죄송합니다ㅠㅠ 가독성이 더 좋다고 생각했었습니다